doc/plots.rst

```diff
@@ -0,0 +1,115 @@
+This means that when switching to another console, the plots from your previous one will disappear, showing you a blank pane.
+
+.. image:: images/plots/plots-switch.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing switching consoles
+
+|
+
+
+
+============
+Options Menu
+============
+
+The options menu in the top right of the :guilabel:`Plots` pane offers several options to customize your plots visualization.
+
+
+~~~~~~~~~~~~~~~~~~~~
+Mute inline plotting
+~~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Mute inline plotting` option is enabled by default, preventing the figures from also appearing in the Console. 
+If you deactivate this option, the plots are generated will appear in the Console every time you run your code.
+
+.. image:: images/plots/plots-inline-plotting.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing plots in the console
+
+|
+
+
+~~~~~~~~~~~~~~~~~
+Show plot outline
+~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Show plot outline` option shows an outline box surrounding the area of the plot, with which you can export the plots.
+
+.. image:: images/plots/plots-outline.gif
+   :align: center
+   :alt: Spyder's plots pane showing a plot's outline
+
+|
+
+
+~~~~~~~~~~~~~~~~~~~
+Fit plots to window
+~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Fit plots to Window` option is also enabled by default making the figures' size to match the size of the pane's window. 
+Disabling it will allow you to zoom the plots in or out using the Zoom buttons at the top of the pane.
+
+.. image:: images/plots/plots-zoom.gif
+   :align: center
+   :alt: Spyder main window showing plots zoom
+
+|
+
+
+
+===============
+Toolbar Options
+===============
+
+On top of the Plots pane you will find the toolbar, which provides several useful features that allow you to interact with the plots. 
+For example, you can navigate through the plots in the pane visible in the thumbnails by using the arrows.
+
+.. image:: images/plots/plots-arrows.gif
+   :align: center
+   :alt: Spyder main window showing plot navigation using arrows.
+
+|
+
+You can also save one or all the plots in the pane as PNGs by clicking the "Save" icon in the toolbar.
+
+.. image:: images/plots/plots-save.gif
+   :align: center
+   :alt: Spyder plots pane showing saving a plot.
+
+|
+
+Additionally, if you want to use a plot in another document, you can click the “Copy to Clipboard” button and paste your plot wherever you want, such as a word processor.
+
+.. image:: images/plots/plots-copy.gif
+   :align: center
+   :alt: Spyder Plots pane showing pasting plot in LibreOffice.
+
+|
+
+Finally, you can use the "Remove" buttons in the Toolbar to delete one or all the plots in the pane.
+
+.. image:: images/plots/plots-remove.gif
+   :align: center
+   :alt: Spyder plots pane showing pasting plot in LibreOffice.
+
+|
+
+
+~~~~~~~~~~~~~~~~~~
+Related components
+~~~~~~~~~~~~~~~~~~
+
+* :doc:`variableexplorer`
```

Comment:
```suggestion
* :doc:`ipythonconsole`
* :doc:`variableexplorer`
```

You should definitely include the Console here, since it is much more linked to the plots pane than the Variable Explorer is.